app.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core
+from os.path import join, dirname
+from dotenv import load_dotenv
+from os import getenv as env
+
+from zoom_recording_ingester_cdk.zoom_recording_ingester_cdk_stack \
```

Comment:
this is kinda ugly. is this normal for python cdk projects? i'm used to the constructs being in a `./lib` directory. I know it's just aesthetics but would love for this to be something like `from cdk import ZoomRecordingIngesterCdkStack`. Rename the package directory and drop an `__init__.py` in there that imports the stack class?